### PR TITLE
[DG] ADY-18 can get shareholder mapping when updating a shop. Delta time to 1 minute and process docs after shops

### PR DESCRIPTION
**Pull request by pandemicGander**

### CODE REVIEW

File: src/main/resources/config/application.yml [hunk 1]
```diff
@@ -112,8 +112,7 @@ jhipster:
 # ===================================================================
 
 application:
-    shopUpdaterCron: "0 */5 * * * ?"
-    docsUpdaterCron: "0 */5 * * * ?"
+    miraklPullCron: "0 */1 * * * ?"
```
Comment (rmcotton) [inline src/main/resources/config/application.yml]: can we set the prod setting to be 5 minutes.

that way we dont break mirakl SLA's

File: src/main/java/com/adyen/mirakl/service/UboService.java [hunk 2]
```diff
@@ -100,9 +103,27 @@
...
-    private void addShareholderCode(final MiraklShop shop, final Integer uboNumber, final ShareholderContact shareholderContact) {
-        shareholderMappingRepository.findOneByMiraklShopIdAndMiraklUboNumber(shop.getId(), uboNumber)
-            .ifPresent(shareholderMapping -> shareholderContact.setShareholderCode(shareholderMapping.getAdyenShareholderCode()));
+    public List<ShareholderContact> extractUbos(final MiraklShop shop, Integer maxUbos) {
+        return extractUbos(shop, null, maxUbos);
+    }
+
+    private void addShareholderCode(final MiraklShop shop, final Integer uboNumber, final ShareholderContact shareholderContact, final GetAccountHolderResponse existingAccountHolder) {
+        final Optional<ShareholderMapping> mapping = shareholderMappingRepository.findOneByMiraklShopIdAndMiraklUboNumber(shop.getId(), uboNumber);
+        if (!mapping.isPresent() && existingAccountHolder != null && existingAccountHolder.getAccountHolderDetails() != null
```
Comment (rmcotton) [inline src/main/java/com/adyen/mirakl/service/UboService.java]: well i'll leave it to sonar.

File: src/main/resources/config/application.yml (hunk 1 above)
Comment (pandemicGander) [reply src/main/resources/config/application.yml]: sure, but demos run on heroku use the prod profile

File: src/main/java/com/adyen/mirakl/service/UboService.java (hunk 2 above)
Comment (pandemicGander) [reply src/main/java/com/adyen/mirakl/service/UboService.java]: sonar seems ok

**Comment by pandemicGander** [comment]
Will resubmit on feature branch with review comments

**Pull request closed without merging**